gambitph/Stackable
# Feat: Added layout switcher

These blocks produce incorrect designs when switching layouts. These will need their variation attributes to be fixed:
- [x] Blockquote
- [x] Call to action
- [x] Card
- [x] Feature grid
- [x] Hero
- [x] Posts
- [x] Pricing box
- [x] Team member
- [x] Testimonial

Process of checking:
- Switch to a layout
- save
- refresh editor (should not error)
- repeat

If there's an error, check if needs:
- `contentAlign: 'center'`
- `hasContainer: true`
